src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -365,7 +373,7 @@ def main():
     job_dir = task_mgr.job_dir
     _create_job_dir(job_dir, destroy_old_job_dir=args.force)
 
-    image_str = _fetch_image(REGISTRY, tool, job_dir, tag=args.image_tag, force_pull=not args.use_cached_image)
```

Comment:
Similarly I'm wondering if the use cached image flag is still needed since we're now tagging the images with an explicit version, so we should always use that version and the exact image it tags. As such either it's on the system or it isn't and we don't need to worry about the cache, right...?

Reply:
👍

Reply:
:+1: